apps/ledger-live-mobile/src/screens/OperationDetails/index.tsx

```diff
@@ -6,8 +6,12 @@ import {
   getDefaultExplorerView,
```

Comment:
why this file needs to be changed?

Reply:
To fix some coding style. Add a blank line between imports declaration and first lines of code (component declaration).
_The Boy Scouts rule_
It's fine by me

Reply:
It's because I pass new props to the children components